roles/bootstrap/templates/install_conda.sh

```diff
@@ -6,4 +6,4 @@ if [ ! -d "/anaconda" ]; then
 fi
 
 /anaconda/bin/pip install findspark
-/anaconda/bin/conda install -y seaborn
\ No newline at end of file
+/anaconda/bin/conda install -y seaborn pandas pyspark
```

Comment:
Installing pyspark like this is dangerous. If it installs a version different than the workers (it just happened this week) it won't work